addons/metallb/20_metallb-config.yaml

```diff
@@ -0,0 +1,24 @@
+{{- range $ipamPool, $allocation := .Cluster.Network.IPAMAllocations }}
+---
```

Comment:
arent't you missing start of a loop here? Previously you had something like this there: `{{- range $ipamPool, $allocation := .Cluster.Network.IPAMAllocations }}`